GOVERNANCE.md

```diff
@@ -1,3 +1,246 @@
-## Kuadrant Governance
+# Steering Committee Governance
 
-Governance for the Kuadrant project is still a work in progress. You can participate in discussions via our project [slack channel](https://kubernetes.slack.com/archives/C05J0D0V525).
+The Kuadrant Steering Committee is the governing body of the Kuadrant project,
+providing decision-making and oversight pertaining to the project bylaws,
+sub-organizations, and financial planning. The Steering Committee also defines
+the project values and structure.
+
+This governance is an open, living document, and will continue to
+evolve as the community and project change.
+
+## Values
+
+The Kuadrant and its leadership embrace the following values:
+
+* Openness: Communication and decision-making happens in the open and is discoverable for future
+  reference. As much as possible, all discussions and work take place in public
+  forums and open repositories.
+
+* Fairness: All stakeholders have the opportunity to provide feedback and submit
+  contributions, which will be considered on their merits.
+
+* Community over Product or Company: Sustaining and growing our community takes
+  priority over shipping code or sponsors' organizational goals.  Each
+  contributor participates in the project as an individual.
+
+* Inclusivity: We innovate through different perspectives and skill sets, which
+  can only be accomplished in a welcoming and respectful environment.
+
+* Participation: Responsibilities within the project are earned through
+  participation, and there is a clear path up the contributor ladder into leadership
+  positions.
+
+## Charter
+
+The following responsibilities and powers belong to the Steering Committee.
+
+Technical governance is expected to be performed by the Maintainers of the
+various project areas and subprojects, as defined in the appropriate OWNERS files.
+
+* Delegate ownership of, responsibility for and authority over areas of the
+  project to specific entities.
+* Define, evolve, and defend the vision, mission and the values of the project.
+* Charter and refine policy for defining new community groups, and establish
+  transparency and accountability policies for such groups
+* Define and evolve project and group governance structures and policies.
+* Act as a final escalation point for disputes in any project repository or
+  group.
+* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
+* Coordinate with the CNCF regarding usage of the Kuadrant brand and deciding
+  project scope, core requirements, and conformance, as well as how that brand
+  can be used in relation to other efforts or vendors.
```

Comment:
I reckon we should be cautious about these 2 points... Maybe we need guidance on how to proceed (?)

Reply:
Those are both things that the CNCF asks the project to decide on.  In the absence of other governance, they ask the "maintainers".

As a CNCF project, Kuadrant is entitled to, with the approval of the TOC/GB (depending), spend CNCF funds on needed things.  As an example, the CNCF pays for Buttondown and Boost subscriptions for Kubernetes.  Someone has to have the authority to request this funding.

Reply:
For that matter, if a vendor wants to have an "official Kuadrant distribution" then that would depend on a program of certification owned by the steering committee.